index.php

```diff
@@ -107,6 +107,27 @@ function failed($reason) {
     failed('sitedata not readable');
 }
 
+define('ABORT_AFTER_CONFIG_CANCEL', true);
```

Comment:
this line is already at the top

Reply:
That's the `ABORT_AFTER_CONFIG`, sans the CANCEL part. I had to move the `ABORT_AFTER_CONFIG_CANCEL` out of the `$fullcheck` conditional on line 161 otherwise I couldn't access the curl class.